packages/pieces/src/lib/apps/google-calendar/triggers/calendar-event.ts

```diff
@@ -0,0 +1,71 @@
+import { OAuth2PropertyValue } from '../../../framework/property';
+import {
+  createTrigger,
+  TriggerStrategy,
+} from '../../../framework/trigger/trigger';
+import { googleCalendarCommon } from '../common';
+import { getLatestEvent, stopWatchEvent, watchEvent } from '../common/helper';
+import { GoogleWatchResponse } from '../common/types';
+
+export const calendarEventUpdatedOrCreatedOrDeleted = createTrigger({
+  // docs: https://developers.google.com/calendar/api/guides/push
+  name: 'new_event_added',
+  displayName: 'New Event',
```

Comment:
I think renaming should be done here

Reply:
I think it should be changed or `new_or_updated_event` , since it capture everything from add / update and deleting